example.py

```diff
@@ -39,10 +40,16 @@ async def cli():
 
 
 async def test_many():
-    device_options = [
-        aioshelly.ConnectionOptions("192.168.1.165", "admin", "test-password"),
-        aioshelly.ConnectionOptions("192.168.1.168"),
-    ]
+    with open("devices.json") as j:
```

Comment:
add this file to `.gitignore`

Reply:
I though to have one in the repo as sample for users

Reply:
We can do both. Add a default, then add it to gitignore to ignore future updates.